careers/technical-engineer.md

```diff
@@ -16,63 +16,67 @@ The senior engineer displays a mastery of the core competencies of software engi
 
 **Collaboration:** They ensure that they do not become a single point of failure for their domain by mentoring and training others. When relevant, they work effectively with others outside the organization (whether in other departments or with vendors). They are capable of effectively running projects of any size independently.
 
+### Responsibilities
+
+* Provide technical leadership and guidance across the product and engineering organizations.
+* Make key architectural decisions and ensure the scalability, performance, and reliability of critical systems.
```

Comment:
I like this, I'd just want to reflect this snippet from the "Technical Knowledge" section above: 

> They take responsibility for the overall health and quality of their domain.

How's this read to you? 

> Own architectural decisions and their consequences, taking responsibility for the holistic health, scalability, and reliability of critical systems over their entire lifecycle.

Reply:
Working through it, I found this one to be mostly redundant alongside the "Take ownership of significant..." below, so I combined them.